src/views/app_panel.dot

```diff
@@ -9,7 +9,7 @@
       {{= this.categoryPanel.render() }}
     {{?}}
   </div>
-  {{= this.servicePanel.render() }}
```

Comment:
Instead of calling the rendering the panel even if it's hidden, just prepare a container in the DOM so that we can render a React component tree when needed.